pkg/configuration/config.go

```diff
@@ -324,25 +290,17 @@ func parseScope(scope string) (string, error) {
 
 // IsValidConfigOption whether the specified key is a valid option
 func IsValidConfigOption(key string) bool {
-	return key == "token" || key == "project" || key == "config" || key == "api-host" || key == "verify-tls"
+	return key == "token" || key == "project" || key == "config" || key == "api-host" ||
```

Comment:
Can we put this those strings in a set and check if the key exists in the set? I think that would be much cleaner and easier to extend in the future.

Reply:
This is the equivalent code in Go. I don't think it's worth it:

```golang
	configOptions := [...]string{
		"token",
		"project",
		"config",
		"api-host",
		"dashboard-host",
		"verify-tls",
	}

	for _, option := range configOptions {
		if key == option {
			return true
		}
	}

	return false
```

Reply:
What about:

``` go
configOptions := [...]string{
		"token",
		"project",
		"config",
		"api-host",
		"dashboard-host",
		"verify-tls",
}

val, exists := dict["foo"]
return exists
```

Reference
https://stackoverflow.com/questions/2050391/how-to-check-if-a-map-contains-a-key-in-go

Reply:
+1